samples/spring-security-hybrid-usage/src/main/java/sample/spring/security/SecurityConfiguration.java

```diff
@@ -82,17 +80,4 @@ private Collection<GrantedAuthority> deriveAuthoritiesFromGroup(Jwt jwt) {
             return groupAuthorities;
         }
     }
-
-    /**
-     * Workaround for IAS only use case until Cloud Authorization Service is globally available.
-     */
-    class MyCustomIasTokenAuthenticationConverter implements Converter<Jwt, AbstractAuthenticationToken> {
```

Comment:
I removed this because it appears to be dead code:
![image](https://user-images.githubusercontent.com/123368068/225903217-2bf37590-25a7-4a0a-9570-2bfc200ea756.png)